rpc/client/client_test.go

```diff
@@ -537,6 +536,63 @@ func TestUnconfirmedTxsLimit(t *testing.T) {
 	assert.NotContains(txRes.Txs, tx2)
 }
 
+func TestEventIndexing(t *testing.T) {
+	assert := assert.New(t)
+	require := require.New(t)
+
+	mockApp := &mocks.Application{}
+	mockApp.On("InitChain", mock.Anything).Return(abci.ResponseInitChain{})
```

Comment:
Use a constant rather than the literal `"InitChain"` and the rest of the literals in this test. I imagine this is an issue with the rest of the tests too, so maybe file a good first issue to track improving this.

Reply:
Issue #349 created.